rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -3279,7 +3279,8 @@ static void message_info_from_sample_info(
     sizeof(info.publication_handle));
   message_info->source_timestamp = info.source_timestamp;
   // TODO(iluetkeb) add received timestamp, when implemented by Cyclone
```

Comment:
I'm going to suggest we update this TODO comment, something like:

```suggestion
  // TODO(iluetkeb) get received_timestamp from Cyclone when implemented there
```

Reply:
Fixed.